serverlessworkflow/modules/ROOT/pages/cloud/operator/developing-workflows.adoc

```diff
@@ -201,6 +208,23 @@ In order to remove the {product_name} Greeting application, you can execute the
 kubectl delete -f <your_file> -n <your_namespace>
 ----
 
+== Referencing resources in the workflow
+
+See the xref:cloud/operator/referencing-resource-files.adoc[] guide for more complex scenarios where you may need to reference other resources in the workflow definition.
+
+== Using another workflow base image
```

Comment:
@ricardozanini Is there a reason why the user would want to do this? It might be worth calling out 1 or 2 of the main use-cases that would prompt a user to consider a custome workflow base image to help people better use the documentation.

Reply:
Yeah, I'll expand this into a security/policy use case where users need to change the image to fit their needs.